### DISCUSSION TITLE
v0.5.5

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#203] [New Releases]
## What's Changed
* Add ability to manage/edit recorded macros
* Add very basic hex viewer
* Windows Installer detects running application
* Open files in read-only mode

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.5.4...v0.5.5

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.5.5'>v0.5.5</a>.</em>

**Comment by Jiang-Chan** [comment]
When to support encoding format selection，I want to use utf-8

**Reply by dail8859** [reply]
I _think_ it should handle some basic UTF-8 encodings, but honestly I'm not sure. I am not overly familiar with various text encodings and how to properly test all the uses cases. Multiple encodings is definitely something I'd like to see in the application at some point.

**Comment by cute-angelia** [comment]
cant set default app for txt 

mac m1
